code/modules/surgery/healing.dm

```diff
@@ -72,6 +71,7 @@
 		urhealedamt_burn *= 0.55
 		umsg += " as best as you can while they have clothing on"
 		tmsg += " as best as they can while [target] has clothing on"
+	experience_given = CEILING(((urhealedamt_burn+urhealedamt_burn)/MEDICAL_SKILL_EASY),1)
```

Comment:
you put burn twice

Reply:
```suggestion
	experience_given = CEILING(((urhealedamt_brute+urhealedamt_burn)/MEDICAL_SKILL_EASY),1)
```

Reply:
Maybe make it use actual user scill level instead of MEDICAL_SKILL_EASY?
More prfficient medick cant aquire many xp from something that it make with closed eyes.

Reply:
frikken atom autocomplete

Reply:
can you write this in ascii